Flat/src/ModulesBTagSFs.cc

```diff
@@ -179,3 +179,40 @@ void PandaAnalyzer::JetBtagSFs()
     tr->TriggerEvent("ak4 gen-matching");
 }
 
+void PandaAnalyzer::JetCMVAWeights() 
+{
+  for (unsigned iShift=0; iShift<17; iShift++) {
+    GeneralTree::csvShift theShift = gt->csvShifts[iShift];
+    gt->sf_csvWeights[theShift] = 1;
+  }
+  if (centralJets.size() < 1) return;
+
+  //get vectors of jet properties
+  std::vector<double> jetPts, jetEtas, jetCSVs, jetCMVAs;
```

Comment:
Can you reserve them to `centralJets.size()` ?

Reply:
yeah